### PR TITLE
GH-86 Added compatibility with Spring Boot 2.6.0

### PR DESCRIPTION
**Pull request by JiHyunSim**
Added compatibility to Spring Boot v2.6.0

### CODE REVIEW

File: spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java [hunk 1]
```diff
@@ -8,28 +8,17 @@
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.amqp.rabbit.config.SimpleRabbitListenerContainerFactory;
-import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
-import org.springframework.amqp.rabbit.connection.ConnectionFactory;
-import org.springframework.amqp.rabbit.connection.ConnectionNameStrategy;
-import org.springframework.amqp.rabbit.connection.RoutingConnectionFactory;
-import org.springframework.amqp.rabbit.connection.SimpleResourceHolder;
-import org.springframework.amqp.rabbit.connection.SimpleRoutingConnectionFactory;
+import org.springframework.amqp.rabbit.connection.*;
```
Comment (rwanderc) [inline spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java:11]: Pls, keep the imports ungrouped

File: spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java [hunk 2]
```diff
@@ -8,28 +8,17 @@
...
 import org.springframework.beans.factory.ObjectProvider;
 import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
 import org.springframework.context.ApplicationContext;
 import org.springframework.core.io.ResourceLoader;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (rwanderc) [inline spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java:18]: Pls, keep the imports ungrouped

Comment (rwanderc) [inline spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java:18]: Pls, keep the imports ungrouped

File: spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java [hunk 3]
```diff
@@ -8,28 +8,17 @@
...
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```
Comment (rwanderc) [inline spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java:21]: Pls, keep the imports ungrouped

Comment (rwanderc) [inline spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java:21]: Pls, keep the imports ungrouped

**Review by rwanderc — CHANGES_REQUESTED** [review]
Thank you for your contribution @JiHyunSim. It looks good overall, but you missed paying attention to the checkstyle issues (imports, missing spaces, lengthy lines). It's not building (run `mvn clean install`) due to these errors.
Could you pls fix them and update the PR?

File: spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java (hunk 2 above)
Comment (JiHyunSim) [reply spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java:18]: ok

Comment (JiHyunSim) [reply spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java:18]: ok

File: spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java (hunk 3 above)
Comment (JiHyunSim) [reply spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java:21]: ok

Comment (JiHyunSim) [reply spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java:21]: ok

**Comment by rwanderc** [comment]
@JiHyunSim thank you for the changes.
But the build is still broken.

**Review by rwanderc — CHANGES_REQUESTED** [review]
Needs the fix of the build

**Review by rwanderc — APPROVED** [review]

**Pull request merged**